src/main/webapp/scripts/ua_kuali_application.js

```diff
@@ -73,7 +74,7 @@ function loadLeadUnitName(unitNumberFieldName, leadUnitNameFieldName ) {
 			},
 			errorHandler:function( errorMessage ) {
 				window.status = errorMessage;
-				setRecipientValue( leadUnitNameFieldName, wrapError( "not found" ), true );
+				//setRecipientValue( leadUnitNameFieldName, wrapError( "not found" ), true );
```

Comment:
This function affects cross-application functionality, I don't think it's a good idea to comment out the error when the unit was not found.